demo/NUnitTestDemo/ShouldAttribute.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace NUnitTestDemo
+{
+    public class ShouldAttribute : PropertyAttribute
+    {
+        public ShouldAttribute(string result) : base(result) { }
+    }
+}
```

Comment:
I had it that way originally, as custom category, and then switched to a property, primarily because of how VS displays it. For categories, you get Category [ShouldPass], etc.

I could certainly make separate property attributes for each outcome.

BTW, I never thought of this as a model for other people, but I suppose it should be taken that way. I use it as my primary acceptance test for the adapter.